### ISSUE TITLE
Merge script does not deal well with non-ascii names of reviewers

### ISSUE THREAD

**Issue by eolivelli** [apache/bookkeeper#241]
- What did you do?
There is an error in the bk-merge-pr.py script in case of reviewers with non-ascii characters in name

- What did you expect to see?
the merge script should run without errors

- What did you see instead?
see comments in #239 from @jiazhai 

```
(master) python dev/bk-merge-pr.py
git rev-parse --abbrev-ref HEAD
Which pull request would you like to merge? (e.g. 34): 240
Traceback (most recent call last):
  File "dev/bk-merge-pr.py", line 555, in <module>
    main()
  File "dev/bk-merge-pr.py", line 466, in main
    pr_reviewers = get_reviewers(pr_num)
  File "dev/bk-merge-pr.py", line 450, in get_reviewers
    reviewers_emails.append('{0} <{1}>'.format(username, useremail))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe1' in position 1: ordinal not in range(128)
```

**Comment by jiazhai** [comment]
would like to have a try to handle it.

**Comment by eolivelli** [comment]
thank you @jiazhai 
I think that you can play with #234, that comes from @sigee, without actually pushing it

**Comment by jiazhai** [comment]
@eolivelli, thanks a lot for the info, have a try on 234 with a fix, and it passed:
```
git:(issue_241) python dev/bk-merge-pr.py
git rev-parse --abbrev-ref HEAD
Which pull request would you like to merge? (e.g. 34): 234
Commit title [Issue 230: Add Checkstyle to the build process (Part 2)]: 
I've re-written the title as follows to match the standard format:
Original: Issue 230: Add Checkstyle to the build process (Part 2)
Modified: ISSUE #230: Add Checkstyle to the build process (Part 2)
Would you like to use the modified title? (y/n): y
Using modified title:
ISSUE #230: Add Checkstyle to the build process (Part 2)

=== Pull Request #234 ===
PR title	Issue 230: Add Checkstyle to the build process (Part 2)
Commit title	ISSUE #230: Add Checkstyle to the build process (Part 2)
Source		sigee/checkstyle2
Target		master
URL		https://api.github.com/repos/apache/bookkeeper/pulls/234

Proceed with merging pull request #234? (y/n): y
git fetch apache-github pull/234/head:PR_TOOL_MERGE_PR_234
remote: Counting objects: 87, done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 87 (delta 69), reused 87 (delta 69), pack-reused 0
Unpacking objects: 100% (87/87), done.
...
```

**Comment by eolivelli** [comment]
@jiazhai  I am checking your patch. As soon as I am OK I think we can merge without @sijie it is ery trivial

**Comment by eolivelli** [comment]
@jiazhai I am sorry the testcase is not usefull, the bug is on reviewers and on on the 'author', I am loking for a good test case

**Comment by eolivelli** [comment]
closed by merging #242

**Comment by sijie** [comment]
@eolivelli @jiazhai - it might be worth opening an issue to make the script handle non-ascii strings extracted from user/committer info.
